lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1627,7 +1627,7 @@ namespace Js
             // We do not own the memory passed into DefaultLoadScriptUtf8. We need to save it so we copy the memory.
             if (*ppSourceInfo == nullptr)
             {
-                *ppSourceInfo = Utf8SourceInfo::New(this, script, parser->GetSourceIchLim(), cb, pSrcInfo, isLibraryCode);
```

Comment:
Why is a bug? if not length is always guaranteed to be same as parser->GetSourceIchLim()? can we assert